packages/langium/src/utils/stream.ts

```diff
@@ -125,6 +125,12 @@ export interface Stream<T> extends Iterable<T> {
     filter<S extends T>(predicate: (value: T) => value is S): Stream<S>;
     filter(predicate: (value: T) => unknown): Stream<T>;
 
+    /**
+     * Returns the elements of the stream that are _non-nullable_, which means they are neither `undefined`
+     * nor `null`.
+     */
+    nonNullable(): Stream<NonNullable<T>>;
```

Comment:
I'd propose to call this method `filterNullable` or `filterNull`

Reply:
I prefer `nonNullable` because it's immediately clear what it does: give you the non-nullable elements. With `filterNullable` or `filterNull` it's not so clear because it's missing the negation. `filterNonNullable` would work, but IMO that's too long.

What do others think @langium/committers?

Reply:
All options are fine me as long as the documentation is clear, which is the case.

Reply:
Moving forward with this for now.